p2p/server.go

```diff
@@ -601,13 +605,28 @@ func (srv *Server) setupDiscovery() error {
 			unhandled = make(chan discover.ReadPacket, 100)
 			sconn = &sharedUDPConn{conn, unhandled}
 		}
+		f := func(r *enr.Record) bool {
+			if srv.forkFilter != nil {
```

Comment:
Inverting the if condition here also helps with readability as the logic block will no be indented anymore